### ISSUE TITLE
Metadata on ended shows is modified on daily show update

### ISSUE THREAD

**Issue by aMvEL** [SickGear/SickGear#1333]
#### Branch version commit hash: 8bedd1c

Expectation | Result
------------ | -------------
Metadata to be mostly untouched when show is ended, except when doing forced Full Update (And possibly if fetched metadata  differs from what is in nfo?) | Episode metadata .nfo files gets modified on daily update, as long as show status is ended. 

**Steps to reproduce:**
*1)* Add an ended show to SG. 
*2)* Creation of episode metadata is activated, in my case target: Kodi.
*3)* Process a file/episode.
*4)* Wait for or Force a daily show update, and observe that the episode .nfo is modified (date modified)

Note that this does not happen to a show that is not ended..

An example with a fresh install, with 1 show with 1 episode.
https://gist.github.com/aMvEL/9bb7b6b10c67e6d1c35e405f2e15cdca

Additional notes:
For all I know, this is intended function in SG. However, it leads to longer library scans in Emby/Kodi now that I added some old Shows to update in SG. I usually remove the shows from SG when they are complete in my library, so I hadn't noticed this before.

If this is intended, please ignore this issue and close it.

If it is not intended, It may be because the refreshShow call in line 194 in show_updater.py sets force to true.

**Comment by JackDandy** [comment]
Metadata .nfo can change on ended shows due to corrections at tv info source BUT...

> it leads to longer library scans in Emby/Kodi now that I added some old Shows to update in SG

Have you been able to prove this? I've also seen longer library update scans on Kodi and have been wondering for months why this is the case, are you sure that an updated .nfo file stamp date produces this result ?

**Comment by aMvEL** [comment]
I can at least confirm that the 3 shows that I currently have in my "production" SG version, that are Ended, gets processed by the Emby for Kodi, as new update, every day  I start Kodi on my HTPC.

Also, Emby logs report changes for those 3 files right after the SG daily show updates

log messages like this for all episodes on those specific shows:
"EpisodeNfoProvider reports change to 124699" etc.etc.

**Comment by JackDandy** [comment]
@aMvEL I've been playing over the last days with tests on this. the latest thing of importance to share with you, is that `Emby for Kodi` is now at 4.1.22 (from repo Kodi Emby Beta Addons 1.0.9). Some issues that I had before, and a lot of its menus under Videos/Addon/Emby that didn't work for me (e.g. TV/TV Recently Added Shows etc) all now work after I did a `Emby/Local DB reset", rebooted, then did all the new prompts and using "Native/Direct playback mode" (not stream). I'm going to play some more with this and see if I see an issue with these .nfo files and any effect on syncing with this version before looking deeper into SG. Can you do the same, and report back. Thanks.

**Comment by aMvEL** [comment]
Sorry for not getting back to you sooner... but crazy busy the last week.

As for emby for kodi, I am already using 4.1.22. I am always on the beta latest for that, as it previously had many issues that I needed the latest beta version to fix.

As for the issue, the only thing I really can say is that on my system, I only get emby for kodi updates for ended shows that actually is added to SG, and as soon as I remove them from SG they do not get update notices in Emby. I have tried local db resets and all that jazz, but no change.
But as I said, it's not really a big issue for me since I have made it regular practice to remove ended shows from SG when I get it archived in my preferred quality. And since you couldn't reproduce as I do, I am inclined to believe it's something wrong somewhere other than SG.

But I really appreciate you looking into this needle in a haystack.
